test/UnitTests/Microsoft.Testing.Extensions.VSTestBridge.UnitTests/ObjectModel/ObjectModelConvertersTests.cs

```diff
@@ -141,7 +141,7 @@ public void ToTestNode_WhenTestResultHasFullyQualifiedTypeAndTrxEnabled_TestNode
 
         var testNode = testResult.ToTestNode(true, TestClient);
 
-        Assert.AreEqual(testNode.Properties.OfType<TrxExceptionProperty>()?.Length, 1);
+        Assert.AreEqual(testNode.Properties.OfType<TrxExceptionProperty>().Length, 1);
```

Comment:
After the removal of `?`, this is now producing:

> test/UnitTests/Microsoft.Testing.Extensions.VSTestBridge.UnitTests/ObjectModel/ObjectModelConvertersTests.cs(144,9): error MSTEST0017: (NETCORE_ENGINEERING_TELEMETRY=Build) Assertion arguments should be passed in the correct order. 'actual' and 'expected'/'notExpected' arguments have been swapped. (https://learn.microsoft.com/dotnet/core/testing/mstest-analyzers/mstest0017)

I believe this is a false negative case of the analyzer. I will open an issue to track this. Meanwhile, you can safely swap the order.